components/servo/Cargo.lock

```diff
@@ -183,6 +183,11 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "bitflags"
+version = "0.4.0"
```

Comment:
Ok. I'm fine with extra bitflags then. Can we file a bug upstream to update it at least?